lib/sprockets/exporters/zlib_exporter.rb

```diff
@@ -0,0 +1,33 @@
+require 'sprockets/exporters/base'
+require 'sprockets/utils/gzip'
+
+module Sprockets
+  module Exporters
+    # Generates a `.gz` file using the zlib algorithm built into
+    # Ruby's standard library.
+    class ZlibExporter < Exporters::Base
+      def setup
+        @gzip_target = "#{ target }.gz"
```

Comment:
I like it, makes it more readable for me. Started doing it after rails style inline blocks/procs.